README.md

```diff
@@ -59,9 +63,14 @@ unless a different image is listed in the metadata.
  :many-others ["see the examples" "creative uses" "visual variety"]}
 ```
 
-**Optional:** [Setup your editor with Clay shortcuts](https://scicloj.github.io/clay/#setup) to visualize the notebook as you write.
+## Preview the Website **(Optional)**
+
+[Set up your editor with Clay REPL commands](https://scicloj.github.io/clay/#setup) to visualize the notebook as you write.
```

Comment:
Should this still say `notebook` or should it be changed to something else? (`namespace`, `article`?)

(there are still several references to `notebook` below)